client/cli/src/lib.rs

```diff
@@ -953,6 +1025,37 @@ impl FedimintCli {
                     .await?;
                 Ok(CliOutput::EpochCount { count })
             }
+            Command::Module { id, arg } => {
+                let cfg = cli.load_config_or_terminate();
+                let decoders = cli.load_decoders(&cfg, &self.module_gens);
+                let db = cli.load_db_or_terminate(&decoders);
+                let (_id, module_cfg) = match id {
+                    ModuleSelector::Id(id) => (
+                        id,
```

Comment:
If this isn't used, maybe don't return it?

Reply:
Then I have to do an extra `.map` to get rid of it on the other case, so this is actually shorter. :D